### PR TITLE
fix(@rjsf-antd): Submit button type bug (#2867)

### LINKED ISSUE #2867: Cannot Submit Form // @rjsf/antd
### Prerequisites

- [X] I have searched the existing issues
- [X] I understand that providing a [SSCCE](http://sscce.org/) example is tremendously useful to the maintainers.
- [X] I have read the [documentation](https://react-jsonschema-form.readthedocs.io/)
- [X] Ideally, I'm providing a [sample JSFiddle](https://jsfiddle.net/n1k0/f2y3fq7L/6/) or a [shared playground link](https://rjsf-team.github.io/react-jsonschema-form/) demonstrating the issue.

### What theme are you using?

antd

### Version

4.2.0

### Current Behavior

When I click submit button,I cannot reach with any response. 
I used default example page

[Example page](https://rjsf-team.github.io/react-jsonschema-form/)
Screen Shot 2022-05-23 at 12 27 00
 

### Expected Behavior

When I click submit button, Form need to be triggered. 

### Steps To Reproduce

1. Set selected theme to antd
2. click submit 
3. Check  logs on console. (nothing appear)

### Environment

_No response_

### Anything else?

_No response_

### PR DESCRIPTION
**Pull request by sarpere**
### Reasons for making this change

Ant design theme assigned with wrong props. This is causing the form to fail to submit. 

I have updated submit button prop; 

from type to HtmlType.

fixes #2867

### CODE REVIEW

File: packages/antd/src/widgets/SubmitButton/SubmitButton.js [hunk 1]
```diff
@@ -5,7 +5,7 @@ const { getSubmitButtonOptions } = utils;
 export default ({ uiSchema }) => {
   const { submitText, norender, props: submitButtonProps }= getSubmitButtonOptions(uiSchema);
   if (norender) {return null;}
-  return (<Button  type="submit" {...submitButtonProps}>
+  return (<Button  htmlType={"submit"} {...submitButtonProps}>
```
Comment (heath-freenome) [inline packages/antd/src/widgets/SubmitButton/SubmitButton.js]: Consider simplifying this to `htmlType="submit"`

**Comment by heath-freenome** [comment]
@sarpere Please revert all of the `package-lock.json` changes as they are not necessary to fix the issue

**Review by heath-freenome — APPROVED** [review]

**Comment by sarpere** [comment]
I have updated the repo.

File: packages/antd/src/widgets/SubmitButton/SubmitButton.js (hunk 1 above)
Comment (sarpere) [reply packages/antd/src/widgets/SubmitButton/SubmitButton.js]: Also I have done this.

File: packages/antd/src/widgets/SubmitButton/SubmitButton.js [hunk 2]
```diff
@@ -5,7 +5,7 @@ const { getSubmitButtonOptions } = utils;
 export default ({ uiSchema }) => {
   const { submitText, norender, props: submitButtonProps }= getSubmitButtonOptions(uiSchema);
   if (norender) {return null;}
-  return (<Button  type="submit" {...submitButtonProps}>
+  return (<Button  htmlType="submit" {...submitButtonProps}>
```
Comment (heath-freenome) [inline packages/antd/src/widgets/SubmitButton/SubmitButton.js:8]: Minor little thing... there seems to be an extra space between `Button` and `htmlType` that you are welcome to remove even though it was there before

Comment (sarpere) [reply packages/antd/src/widgets/SubmitButton/SubmitButton.js]: It's all done. :)

Comment (heath-freenome) [reply packages/antd/src/widgets/SubmitButton/SubmitButton.js]: It look like you are going to have to update the test snapshots as a result of this change

Comment (sarpere) [reply packages/antd/src/widgets/SubmitButton/SubmitButton.js]: I have updated snapshots. You can try to trigger workflow.

**Pull request merged**